src/r2mm/installing/default_installation_rules/InstallationRuleApplicator.ts

```diff
@@ -14,6 +14,10 @@ import InstallRules_BackpackHero from '../default_installation_rules/game_rules/
 import InstallRules_BONELAB from '../default_installation_rules/game_rules/InstallRules_BONELAB';
 import { buildBepInExRules } from '../default_installation_rules/game_rules/InstallRules_BepInex';
 import * as path from 'path';
+import { buildGodotMlRules } from "../default_installation_rules/game_rules/InstallRules_GodotML";
```

Comment:
This would probably read better as `buildGodotModLoaderRules`
Alternatively further down you use `ML` instead of `Ml` so it'd be good for consistency if you prefer ML.